src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -253,9 +258,31 @@ def get_obs_data(bufr, profile_meta_data, record_number=None):
         obs_data[('geoid_height_above_reference_ellipsoid', 'MetaData')] - \
         obs_data[('earth_radius_of_curvature', 'MetaData')]
 
+    if qc:
```

Comment:
yes please just pass `qc` into the function if it needs it

and if you ever use `global` doesn't it come before the variable?